index.html.in

```diff
@@ -334,14 +334,17 @@
                   <form class="form-horizontal" role="form" id="sandboxForm">
                       <fieldset>
                           <legend>
-                              <span data-text="APy_Sandbox_Help">
+                              <span data-text="APy_Sandbox_Help"></span>
                               <a href="http://wiki.apertium.org/wiki/Apertium-apy" target="_blank" class="pull-right" rel="noopener">
                                   <abbr title="Apertium API in Python">APy</abbr>
                                   <i class="fa fa-question-circle"></i>
                               </a>
                           </legend>
                           <div class="form-group">
-                              <label for="sandboxInput" class="col-sm-2 control-label" data-text="APy_Request">APy Request</label>
+                              <label for="sandboxInput" class="col-sm-2 control-label">
+                                  <label data-text="APy_Request"></label>
+                                  <i id="APy_Endpoint" class="fa fa-info-circle" data-toggle="tooltip"></i>
```

Comment:
`APy_Endpoint` -> `APyEndpoint` for consistency. Or even just, `endpoint` is fine, I think.